web/src/js/components/Settings/Widgets/WidgetSettingsComponent.js

```diff
@@ -19,15 +19,22 @@ class WidgetSettings extends React.Component {
   }
 
   componentDidMount() {
-    const { appWidget, widget } = this.props
-
+    const { appWidget, widget, user } = this.props
     var config
     if (widget && widget.config) {
       config = JSON.parse(widget.config)
     }
 
-    const settings = this.getConfig(JSON.parse(appWidget.settings), config)
-
+    let settings = this.getConfig(JSON.parse(appWidget.settings), config)
+    // temporarily adding yahoo as an option for yahoo demo team
+    if (user.id === YAHOO_USER_ID) {
+      settings = settings.map(setting => {
```

Comment:
I didn't bother with figuring out how to modify prod db.  this will only run on yahoo user